README.md

```diff
@@ -69,6 +69,7 @@ The table below outlines what version of Nexus Repository the plugin was built a
 | v1.0.3         | >=3.15.2-01              |
 | v1.0.4         | >=3.18.0-01              |
 | v1.1.1 In product | >=3.20.0              |
+| v1.1.2 In product | >=3.21.0-05              |
```

Comment:
Good here but the line above this seems to be missing the -04.
Might be good to just decide and standardize.

Reply:
exactly) I still not sure what is best. Going to ask Joseph!